### ISSUE TITLE
Enhancement: Support custom response for batch actions

### ISSUE THREAD

**Issue by jowilf** [jowilf/starlette-admin#207]
Enhance the @action decorator to support custom responses, such as redirects or rendering HTML pages, instead of just returning strings (related to #205)

Todo: 
- [ ] Add a custom_response flag to the @action decorator function (default: False).
- [ ] Update submitAction() in [list.js](https://github.com/jowilf/starlette-admin/blob/main/starlette_admin/statics/js/list.js) to handle custom responses.
- [ ] Modify handle_action in the base admin class to support custom responses.
- [ ] Ensure backward compatibility.
- [ ] Include an example in [custom_actions example](https://github.com/jowilf/starlette-admin/tree/main/examples/custom_actions)
- [ ] Add tests.
- [ ] Update the documentation

**Comment by arnabJ** [comment]
I know this is quite old and the custom responses are working properly.

My issue or requirement is a bit different.
I have added a custom action using the @action decorator and inside it I am perform an action. I have wrapped it up inside try/except and I am returning an error messing from the except block. But it still shows a green check-mark instead of a red cross or some other icon indicating failure.

How can I show an error message instead of a success/confirmation message from the except block?

More info:
I tried raising an exception like this:
```python
try:
    ...
except Exception as e:
    raise Exception("Descriptive error message here...")
```

It did showed a red circle with a diagonal line but the message was Something went wrong instead of my text: Descriptive error message here...
